frontend/packages/dev-console/integration-tests/features/monitoring/Changing_context_in_Monitoring_Dashboard_3898

```diff
@@ -0,0 +1,29 @@
+Feature: Changing context in Monitoring Dashboard
+	User should be able I want to change the filter in the Monitoring Dashboard
+
+Background:
+    Given user logged into the openshift application
+    And user is at the developer perspective
+
+@regression
+Scenario: Checking filter dropdown for workloads on Monitoring Dashboard navigating from workloads
```

Comment:
I think scenario description and step looks different

In Scenario description, You mentioned "filter dropdown values verification for workloads in dashboard
But in steps, when statement says refreshes the page..  
and Then statement says workloads auto selection